packages/venia-concept/src/components/ProductStickyFooter/productStickyFooter.css

```diff
@@ -0,0 +1,71 @@
+.root {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 5rem;
+    background-color: white;
+    border-top: 1px solid #ececed;
+}
+
+.addToCart {
+    width: 8.625rem;
+    height: 2rem;
+    background-color: rgb(var(--venia-teal));
+    color: white;
+    border-radius: 100px;
+    outline: none;
+}
+
+.addToCartSuccess {
+    composes: addToCart;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.addToCartInProgress {
+    composes: addToCart;
+    display: flex;
+    justify-content: space-between;
+    border: 1px solid rgb(var(--venia-teal));
+    background-color: white;
+    padding: 0 2.75rem;
+}
+
+@keyframes loading {
+    0% {
+        background-color: #c6e8e9;
+    }
+
+    100% {
+        background-color: #00898d;
+    }
+}
+
+.loadingDot {
+    --size: 0.75rem;
+    background-color: #c6e8e9;
+    width: var(--size);
+    height: var(--size);
+    animation-name: loading;
+    animation-duration: 2s;
+    animation-iteration-count: infinite;
+    animation-direction: reverse;
+    border-radius: var(--size);
+}
+
+.loadingDot_1 {
+    composes: loadingDot;
+    animation-delay: 0.5s;
+}
+
+.loadingDot_2 {
+    composes: loadingDot;
+    animation-delay: 1s;
+}
+
+.loadingDot_3 {
+    composes: loadingDot;
+    animation-delay: 1.5s;
+}
```

Comment:
The loading dots look great!

I'd like to use these outside of just this file too 😉